src/SFA.DAS.Rofjaa.Domain/Interfaces/IAgencyRepository.cs

```diff
@@ -6,7 +6,7 @@ namespace SFA.DAS.Rofjaa.Domain.Interfaces
 {
     public interface IAgencyRepository
     {
-        Task<Agency> Get(long id);
+        Task<Agency> Get(long legalEntityId);
```

Comment:
I don't think we need a repository currently (is this interface actually implemented anywhere?)